seccomp/tracer.go

```diff
@@ -260,9 +256,13 @@ func (t *syscallTracer) seccompSessionDelete(pid uint32) {
 			}
 
 			// if the container is no longer there, or the pid being deleted is the
-			// container's init pid, we close all seccomp fds for that container
+			// container's init pid, we close all seccomp sessions for that container
```

Comment:
Just realized that you added this 'ContainerLookupById' instruction in the previous commit, which reduces a bit the coupling between the container-state logic and the handlers one in terms of performance. That concerned me a bit at first glance coz we already have one of these calls in the syscall-processing code-path -- see process() --, but luckily this other one that you added is executed in the context of the sessionsMonitor thread, so we are not duplicating efforts here.